### DISCUSSION TITLE
“时分秒”中若使用冒号，应采用英文半角还是中文全角？

### DISCUSSION THREAD

**Discussion by CoolSpring8** [sparanoid/chinese-copywriting-guidelines#142] [Q&A] [answered]
这是日常交流中时常困扰我的一个疑问。

- GB/T 15834―2011《标点符号用法》中似乎没有提及冒号的此类用法。
- GB/T 15835―2011《出版物上数字用法》在 5.1.5 中提到“‘时’‘分’也可按照 GB/T 7408―2005 的 5.3.1.1 和 5.3.1.2 中的扩展格式，用‘:’替代。”这里的冒号是半角的。
- GB/T 7408―2005《数据元和交换格式 信息交换 日期和时间表示法》对应节中的冒号亦为半角。

这与全形和半形的用法（“使用全形中文標點”，“數字使用半形字符”，“遇到完整的英文整句、特殊名詞，其內容使用半形標點”）是否会产生矛盾呢？

**Comment by liyishuai** [comment, marked as answer]
时分秒中冒号与数字应视为整体，故建议使用半角。
